data/remnant/remnant missions.txt

```diff
@@ -3445,3 +3445,276 @@ event "remnant: pantica void sprites"
+			choice
+				`	"They know you are coming back, right?"`
+				`	"What was that about?"`
+				`	"Alright, let's get going."`
+					goto astralmoving
+			`	"Well, what we are doing is dangerous. We will be crossing Korath territory, a wide expanse of uninhabited space, and then scanning a black hole. I have confidence that we will return, but nonetheless there is still a chance that we die in the process.`
+			`	"Long ago we, as a people, decided that we must face the fear of separation, the unknown, and the danger. One of the ways we do that is by holding these informal ceremonies. When one of us is going to depart on their first long distance voyage into hostile territory, a group of Remnant gather to bid them farewell. It is a way to strengthen ourselves and remind each other of our connections."`
+			choice
+				`	"Thanks for the explanation."`
+					goto astralmoving
+				`	"Kind of like a funeral, with friends and family gathering to see you off?"`
+			`	"Oh, no. I said my farewells to my friends and relatives before we left. Those who gathered to see me off today were more of a representation of Remnant society. I only recognized a few of them, but they took the time to come down to the <ship> to see me before I left. They came to see me, and wish me well, so that no matter what challenges I face, I will know that the Remnant remember me and appreicate what I am doing." Her gestures fade into contemplative stillness for a moment, then resume motion.`
+			label astralmoving
+			`	She gestures towards the airlock door. "Lead the way, Captain."`
+				accept
+	on stopover
+		conversation
+			`As the energy ripples from the Jump Drive fade, the spectacular void that is Sagittarius A* takes shape before you. You can make out a rough silhouette against the background of stars and the glow of the acretion disk. As you slowly fall towards the singularity, you activate the anti-gravity systems while various alarms start to trigger. While vastly underpowered for dealing with the immense pull of a black hole, they provide a degree of protection while you maintain a safe distance from it.`
+			`	After a few minutes of just absorbing the spectacle before you, Dawn pulls herself away from the cockpit windows and scurries down to the lab. Leaving you to your thoughts, she quickly has the lab set up and running a variety of recordings, test, and observation equipment. After a few hours of work, she reappears on the bridge.`
+			`	"Things are proceeding smoothly, Captain. All the sensors are working nominally and we are creating extensive archives of astronomical data as well. Another few hours and we should have all the data we need."`
```

Comment:
Shouldn't it say normally here not nominaly, it seems and odd word choice
(sorry for comment in main discusion i am new to github)

Reply:
nominal is basically a technical synonym for normal. Very common in aerospace environments. (if you listen to SpaceX launches, you will hear it used in this way frequently.)